src/trix/views/attachment_view.coffee

```diff
@@ -47,7 +47,7 @@ class Trix.AttachmentView extends Trix.ObjectView
       data.trixSerialize = false
 
     if href = @getHref()
-      element = makeElement("a", {href})
+      element = makeElement("a", {href, tabindex: -1})
```

Comment:
Should we set `tabindex` in concert with this instead? https://github.com/basecamp/trix/blob/d4b81b80e23c1a1ef08486f1ec9982808ec66fa6/src/trix/views/attachment_view.coffee#L56

I *think* it's the `contenteditable` attribute that mucks with the tab order. Not `<a>`.